sources/packages/backend/libs/services/src/student/student.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable } from "@nestjs/common";
+import {
+  RecordDataModelService,
+  SINValidation,
+  Student,
+  User,
+} from "@sims/sims-db";
+import { DataSource } from "typeorm";
+import { LoggerService, InjectLogger } from "@sims/utilities/logger";
+
+@Injectable()
+export class StudentService extends RecordDataModelService<Student> {
+  constructor(private readonly dataSource: DataSource) {
+    super(dataSource.getRepository(Student));
+    this.logger.log("[Created]");
+  }
+
+  /**
+   * Gets all the students that have the SIN validation pending.
+   * @returns Students pending SIN validation.
+   */
+  async getStudentsPendingSinValidation(): Promise<Student[]> {
```

Comment:
Methods like this are not part of the integration scope only?

Reply:
Yes. This is what I mentioned this comment. https://github.com/bcgov/SIMS/pull/1624/files#r1058616674